src/css/lib/yasqe.min.css

```diff
@@ -22,7 +22,7 @@
   .yasqe .CodeMirror {
     /* Set height, width, borders, and global font properties here */
     font-family: monospace;
-    height: 330px;
+    height: 300px;
```

Comment:
This css comes from the yasqe library, right? In this case it is better to not modify it, but introduce the styling through our styles, in the case through new-sparql.css. Please, move all yasqe style changes there.